app/models/dictionary.rb

```diff
@@ -245,13 +245,13 @@ def add_patterns(patterns)
     end
   end
 
-  def add_entries(raw_entries, normalizer = nil)
+  def add_entries(raw_entries, analyzer = nil)
```

Comment:
nilだったら `Analyzer.new` にしてしまうと、あとの判定が減らせて楽になりそうです。
```suggestion
  def add_entries(raw_entries, analyzer = Analyzer.new)
```

のイメージです。

Reply:
add_entriesメソッドに加えて、他のコードも可能な限り変更しました。
https://github.com/pubannotation/pubdictionaries/pull/157/commits/15ab48890d0821ed4eff627c2df879ff02ea2db3